### PR TITLE
command-line-flags-for-tidb-configuration: Add missing flags

### PR DESCRIPTION
**Pull request by dveeden**
### What is changed, added or deleted? (Required)

Add missing flags for `tidb-server`.

```
$ diff <(path/to/tidb/bin/tidb-server -h 2>&1 | awk '/^  -/ { print $1 }' | sort) <(awk '/^##/ { print $2 }' command-line-flags-for-tidb-configuration.md | cut -d\` -f2 | sed -e 's/--/-/g' | sort)
3d2
< -alsologtostderr
9d7
< -help
11,12d8
< -initialize-insecure
< -initialize-secure
15,16d10
< -log_backtrace_at
< -log_dir
19d12
< -logtostderr
35d27
< -stderrthreshold
37d28
< -temp-dir
39d29
< -v
41d30
< -vmodule
```

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v6.4 (TiDB 6.4 versions)
- [x] v6.3 (TiDB 6.3 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):
  - https://github.com/pingcap/tidb/pull/28482
  - https://github.com/pingcap/tidb/pull/28487
  - https://github.com/pingcap/tidb/pull/36088

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by tangenta — APPROVED** [review]

File: command-line-flags-for-tidb-configuration.md [hunk 1]
```diff
@@ -196,4 +236,14 @@ When you start the TiDB cluster, you can use command-line options or environment
 ## `--repair-list`
 
 + The names of the tables to be repaired in the repair mode.
-+ Default: `""`
\ No newline at end of file
++ Default: `""`
+
+## `-v`
```
Comment (xhebox) [inline command-line-flags-for-tidb-configuration.md:241]: I think these flags comes from `160:    github.com/golang/glog v1.0.0 // indirect`, and they should not be included: they should not appear in tidb --help at all. This should be a bug.

`--temp-dir` and `--initialize-secure` sounds good to me.

Comment (dveeden) [reply command-line-flags-for-tidb-configuration.md:241]: Looks like this affects all v6.x versions.

```
$ for b in $(ls .tiup/components/tidb/*/tidb-server | grep -v nightly); do echo "# $b"; $b -h 2>&1 | grep '\-vmodule'; done
# .tiup/components/tidb/v4.0.16/tidb-server
# .tiup/components/tidb/v5.0.6/tidb-server
# .tiup/components/tidb/v5.1.0/tidb-server
# .tiup/components/tidb/v5.1.3/tidb-server
# .tiup/components/tidb/v5.1.4/tidb-server
# .tiup/components/tidb/v5.2.4/tidb-server
# .tiup/components/tidb/v5.3.0/tidb-server
# .tiup/components/tidb/v5.3.1/tidb-server
# .tiup/components/tidb/v5.4.0/tidb-server
# .tiup/components/tidb/v5.4.1/tidb-server
# .tiup/components/tidb/v5.4.2/tidb-server
# .tiup/components/tidb/v6.0.0/tidb-server
  -vmodule value
# .tiup/components/tidb/v6.1.0/tidb-server
  -vmodule value
# .tiup/components/tidb/v6.1.1/tidb-server
  -vmodule value
# .tiup/components/tidb/v6.1.2/tidb-server
  -vmodule value
# .tiup/components/tidb/v6.2.0/tidb-server
  -vmodule value
# .tiup/components/tidb/v6.3.0/tidb-server
  -vmodule value
```

Could you file an issue for this? I'll remove it from this PR.

Comment (xhebox) [reply command-line-flags-for-tidb-configuration.md]: Of course, pingcap/tidb#38857.

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**